backend/lib/config/gardener.js

```diff
@@ -55,20 +125,43 @@ function getEnvironmentVariable (env, name, type) {
       return value
   }
 }
+function getValueFromEnvironment (env, envVar, type) {
+  const value = env[envVar]
+  return convertValue(value, type)
+}
+
+function getValueFromFile (filePath, type) {
+  try {
+    if (!fs.existsSync(filePath)) {
+      return undefined
+    }
+    const fileContent = fs.readFileSync(filePath, 'utf8')
+    return convertValue(fileContent, type)
+  } catch (error) {
+    /* ignore */
+  }
```

Comment:
Why do we need two sync calls?
```suggestion
  try {
    const value = fs.readFileSync(filePath, 'utf8')
    return parseConfigValue(value, type)
  } catch (error) {
   return undefined
  }
```

Reply:
✅